src/SQLite.Net/SQLiteCommand.cs

```diff
@@ -488,7 +488,7 @@ private object ReadCol(IDbStatement stmt, int index, ColType type, Type clrType)
                 }
                 else
                 {
-                    value = DateTime.Parse(_sqlitePlatform.SQLiteApi.ColumnText16(stmt, index), CultureInfo.InvariantCulture);
+                    value = DateTime.Parse(_sqlitePlatform.SQLiteApi.ColumnText16(stmt, index), CultureInfo.InvariantCulture).ToUniversalTime();
```

Comment:
better to use `return DateTime.Parse(_sqlitePlatform.SQLiteApi.ColumnText16(stmt, index), CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal)`